### ISSUE TITLE
Autocaptioning

### ISSUE THREAD

**Issue by alcinnz** [elementary/videos#227]
## Problem

Many videos lack captions, which is an accessibility issue. While this is best solved within the video itself, it'd be great if Videos had a fallback available for this too common case.

## Proposal

I propose that an option be added to the "Subtitles" dropdown selection which offers to install & use PocketSphinx or Kaldi as a source for these subtitles. Both these projects include GStreamer elements which can easily be integrated into Videos, which is what prompted me to open this feature request.

## Prior Art

YouTube & I think Vimeo includes autocaptioning.
